From d1c8cb97a72bd30eb15974eeec953316c9846092 Mon Sep 17 00:00:00 2001
From: Robby Cai <R63905@freescale.com>
Date: Wed, 1 Aug 2012 09:05:25 +0800
Subject: [PATCH 339/505] ENGR00218994: vpu: change the DUT header file
 copyright to GPL

Changed the copyright to GPL.
No functional change in this patch.

Signed-off-by: Robby Cai <R63905@freescale.com>
Signed-off-by: Chen Fei <B02249@freescale.com>
---
 test/mxc_vpu_test/dut_api_vts.h    | 24 ++++++++++++++++--------
 test/mxc_vpu_test/dut_probes_vts.h | 24 ++++++++++++++++--------
 2 files changed, 32 insertions(+), 16 deletions(-)

diff --git a/test/mxc_vpu_test/dut_api_vts.h b/test/mxc_vpu_test/dut_api_vts.h
index 2f248ee..ad06ec1 100644
--- a/test/mxc_vpu_test/dut_api_vts.h
+++ b/test/mxc_vpu_test/dut_api_vts.h
@@ -1,17 +1,25 @@
 /*
- ***********************************************************************
- * Copyright 2005-2012 Freescale Semiconductor, Inc.
- * All modifications are confidential and proprietary information
- * of Freescale Semiconductor, Inc. ALL RIGHTS RESERVED.
- ***********************************************************************
+ * Copyright (C) 2005-2012 Freescale Semiconductor, Inc. All Rights Reserved.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307 USA
+ *
  */
 
 /*****************************************************************************
 ** dut_api_vts.h
 **
-** Use of Freescale code is governed by terms and conditions
-** stated in the accompanying licensing statement.
-**
 ** Description: Implements structures used by DUTs
 **              and gives the VTS used APIs of DUTs.
 **
diff --git a/test/mxc_vpu_test/dut_probes_vts.h b/test/mxc_vpu_test/dut_probes_vts.h
index c9f13c9..6ddd184 100644
--- a/test/mxc_vpu_test/dut_probes_vts.h
+++ b/test/mxc_vpu_test/dut_probes_vts.h
@@ -1,17 +1,25 @@
 /*
- ***********************************************************************
- * Copyright 2005-2012 Freescale Semiconductor, Inc.
- * All modifications are confidential and proprietary information
- * of Freescale Semiconductor, Inc. ALL RIGHTS RESERVED.
- ***********************************************************************
+ * Copyright (C) 2005-2012 Freescale Semiconductor, Inc. All Rights Reserved.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307 USA
+ *
  */
 
 /*****************************************************************************
 ** dut_probes_vts.h
 **
-** Use of Freescale code is governed by terms and conditions
-** stated in the accompanying licensing statement.
-**
 ** Description: Implements structures and enumulations used by DUT
 **              when it invokes callback functions.
 **
-- 
1.8.0

